docs/releases/2.1.txt

```diff
@@ -173,8 +173,71 @@ Models
+
+* The new :class:`~django.db.models.functions.Acos` database function returns
+  the arc cosine of an expression returning a numeric value.
+
+* The new :class:`~django.db.models.functions.Asin` database function returns
+  the arc sine of an expression returning a numeric value.
+
+* The new :class:`~django.db.models.functions.Atan` database function returns
+  the arc tangent of an expression returning a numeric value.
+
+* The new :class:`~django.db.models.functions.Atan2` database function returns
+  the arc tangent of two expressions returning a numeric value.
+
+* The new :class:`~django.db.models.functions.Ceiling` database function returns
+  the smallest integer greater than or equal to an expression returning a numeric
+  value.
+
+* The new :class:`~django.db.models.functions.Cos` database function returns
+  the cosine of an expression returning a numeric value.
+
+* The new :class:`~django.db.models.functions.Cot` database function returns
+  the cotangent of an expression returning a numeric value.
+
+* The new :class:`~django.db.models.functions.Degrees` database function returns
+  the degrees of an expression returning a numeric value.
+
+* The new :class:`~django.db.models.functions.Exp` database function returns
+  the the value of e (the base of natural logarithms) raised to the power of an
+  expression returning a numeric value.
+
+* The new :class:`~django.db.models.functions.Floor` database function returns
+  the largest integer value not greater than an expression returning a numeric
+  value.
+
+* The new :class:`~django.db.models.functions.Log` database function returns
+  the natural logarithm of an expression returning a numeric value.
+
+* The new :class:`~django.db.models.functions.Mod` database function returns
+  the module value of two expressions returning a numeric value.
+
+* The new :class:`~django.db.models.functions.PI` database function returns
+  PI(3.141592...) returning a numeric value.
+
+* The new :class:`~django.db.models.functions.Power` database function returns
+  the power of two expressions returning a numeric value.
+
+* The new :class:`~django.db.models.functions.Radians` database function returns
+  the radians of an expression returning a numeric value.
+
+* The new :class:`~django.db.models.functions.Round` database function returns
+  the nearest integer of an expression returning a numeric value.
+
+* The new :class:`~django.db.models.functions.Sin` database function returns
+  the sine of an expression returning a numeric value.
+
+* The new :class:`~django.db.models.functions.Sqrt` database function returns
+  the square root of an expression returning a numeric value.
+
+* The new :class:`~django.db.models.functions.Tan` database function returns
+  the tangent of an expression returning a numeric value.
+
 * The new :class:`~django.db.models.functions.TruncWeek` function truncates
-  :class:`~django.db.models.DateField` and
+  :class:`~django.db.models.DateFielarc cosined` and
```

Comment:
Not sure how you got this in here, but please undo this edit.

Reply:
Thanks. I guess there was a mistake for copy and paste. I didn't notice that. I will undo it.